### DISCUSSION TITLE
Summary page when Backrest isn't taking backups

### DISCUSSION THREAD

**Discussion by averymd** [garethgeorge/backrest#624] [Q&A] [answered]
I'm messing with Backrest as a way to see my restic backups across a bunch of different servers. Even though Backrest can't actually run those backups, I have all the repos configured so I can at least compute stats and run checks. That all seems fine.

The homepage of Backrest--where all repos are listed with theoretical "Bytes Scanned" and "Bytes Added" data--doesn't show any stats, though, even after I compute stats on a repo. Does that only update if Backrest is making the backups itself?

**Comment by garethgeorge** [comment, marked as answer]
Right -- those stats are collected only for backups initiated by backrest.
